content/en/real_user_monitoring/guide/connect-session-replay-to-your-third-party-tools.md

```diff
@@ -0,0 +1,90 @@
+---
+title: Connect Session Replay To Your Third-Party Tools
+kind: guide
+further_reading:
+- link: '/real_user_monitoring/session_replay/'
+  tag: 'Documentation'
+  text: 'Learn about Session Replay'
+---
+
+## Overview
+
+Session Replay provides visual insights to complement user analytics data. If you are using third-party tools for customer experience, website analytics, and more, you can connect them to Session Replay. This guide walks you through how to access the Session Replay URL to use in integrations, live from the browser where the session is taking place. 
+
+## Use cases
+
+You may want to connect a third-party tool with Session Replay for a more comprehensive view of user experience indicators such as the following:
+
+- Form survey results
+- Customer experience tools
+- Data analytics
+
+## Get the Session Replay link
+
+To fetch the URL for the current user session's recording, use the following snippet, depending on the installation method you used to set up RUM:
+
+{{< tabs >}}
+{{% tab "NPM" %}}
+
+```javascript
+import { datadogRum } from '@datadog/browser-rum';
+
+datadogRum.init({
+    ...,
+    // optional, only needed if using a custom domain name
+    subdomain: ''
```

Comment:
Maybe we should add a paragraph explaining why the subdomain is needed, if as a customer you're accessing Datadog via your custom subdomain. This way the links you get from the function call are also customized.

Reply:
Hey @acorretti, thank you for the suggestion, which sounds very helpful. I will create a new PR with this suggestion and send it your way for review :)